tooling/templatize/config/config.go

```diff
@@ -50,19 +54,43 @@ func (cp *configProviderImpl) GetVariables(cloud, deployEnv string) (Variables,
 	return variables, err
 }
 
-func (cp *configProviderImpl) loadConfig() (*VariableOverrides, error) {
-	tmpl, err := template.ParseFiles(cp.config)
+func (cp *configProviderImpl) loadConfig(cloud, deployEnv string) (*VariableOverrides, error) {
+	vars := map[string]interface{}{
+		"ctx": map[string]interface{}{
+			"region":      cp.region,
+			"cloud":       cloud,
+			"deployEnv":   deployEnv,
+			"regionStamp": cp.regionStamp,
+			"cxStamp":     cp.cxStamp,
+		},
+	}
 
-	if err == nil {
-		var tmplBytes bytes.Buffer
+	functions := template.FuncMap{
```

Comment:
FuncMap should be in main.go instead of config.go. Since these functions are only used in local development, not in pipeline